### DISCUSSION TITLE
manjaro problem

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3085] [Q&A]
i install qtile for manjaro but not start
[qtile.log](https://github.com/qtile/qtile/files/7692698/qtile.log)

**Comment by elParaguayo** [comment]
How did you install qtile on Manjaro?

**Reply by ghost** [reply]
sudo pacman -S qtile
and try yay -S qtile

**Reply by elParaguayo** [reply]
Can you do `pacman -Qi qtile` and paste the output here. I wonder if some dependencies are missing (your log suggests there's a problem with `cairo`)

**Reply by ghost** [reply]
ı normaly use arch linux and not problem ı install arch linux sorry

**Reply by elParaguayo** [reply]
No need to apologise. It should work on Manjaro though (unless they've changed the Arch package) so I'm not sure why it didn't.

**Reply by ghost** [reply]
display manager ı select qtile and enter password first black screen and returned display manager ı dont know why

ı try yay -S qtile and python3 -m pip install psutil python-magic pyinstaller  cairocffi cffi xcffib
sudo pacman -S tk

**Reply by elParaguayo** [reply]
Installing via `pacman` should install all the necessary dependencies. I would add that it's best not to use pip to install python packages, ideally you should install these via pacman too as, otherwise, pacman won't know what python packages are installed.

**Reply by hamensman** [reply]
If you're using a display manager like lightdm (like I do), you will need to create a desktop file in `/usr/share/xsessions/qtile.desktop` and write what is written here (if you have't done so already): https://github.com/qtile/qtile/blob/master/resources/qtile.desktop

**Reply by hamensman** [reply]
Last time I installed Arch Linux, it didn't install `python-setuptools`. So if this is still the case, you might want to try `pacman -S qtile python-setuptools`.

And if that still doesn't work, see if `xf86-video-intel` is not installed for whatever reason. Usually on Manjaro, it is. If not, install it.

**Reply by elParaguayo** [reply]
> If you're using a display manager like lightdm (like I do), you will need to create a desktop file in `/usr/share/xsessions/qtile.desktop` and write what is written here (if you have't done so already): https://github.com/qtile/qtile/blob/master/resources/qtile.desktop

The Arch package installs the desktop file to that location: https://github.com/archlinux/svntogit-community/blob/packages/qtile/trunk/PKGBUILD#L92

**Reply by elParaguayo** [reply]
> Last time I installed Arch Linux, it didn't install `python-setuptools`. So if this is still the case, you might want to try `pacman -S qtile python-setuptools`.
> 
> And if that still doesn't work, see if `xf86-video-intel` is not installed for whatever reason. Usually on Manjaro, it is. If not, install it.

That's also included in the package: https://github.com/archlinux/svntogit-community/blob/packages/qtile/trunk/PKGBUILD#L22
